src/utils.ts

```diff
@@ -1292,3 +1293,5 @@ export function promiseWithResolvers<T>() {
   });
   return { promise, resolve, reject } as const;
 }
+
+export const randomBytes = promisify(crypto.randomBytes);
```

Comment:
I am not seeing the crypto import at the top of this file, is this using the global? I'm not sure there's a `randomBytes` on the global and I don't think it exists in Node.js v16

Reply:
nevermind!